confmap/provider/httpprovider/README.md

```diff
@@ -1,13 +1,23 @@
+## Introduction
 What is this new component httpprovider?
-- An implementation of `confmap.Provider` for HTTP (httpprovider) allows OTEL Collector the ability to load configuration for itself by fetching and reading config files stored in HTTP servers.
+- An implementation of `confmap.Provider` for HTTP/HTTPS (httpprovider) allows OTEL Collector the ability to load configuration for itself by fetching and reading config files stored in HTTP servers.
 
-How this new component httpprovider works?
+How does this new component httpprovider work?
 - It will be called by `confmap.Resolver` to load configurations for OTEL Collector.
-- By giving a config URI starting with prefix 'http://', this httpprovider will be used to download config files from given HTTP URIs, and then used the downloaded config files to deploy the OTEL Collector.
+- By giving a config URI starting with prefix 'http://' or 'https://', this httpprovider will be used to download config files from given HTTP URIs, and then used the downloaded config files to deploy the OTEL Collector.
 - In our code, we check the validity scheme and string pattern of HTTP URIs. And also check if there are any problems on config downloading and config deserialization.
 
 Expected URI format:
 - http://...
+- https://...
 
 Prerequistes:
-- Need to setup a HTTP server ahead, which returns with a config files according to the given URI
\ No newline at end of file
+- Need to setup a HTTP server ahead, which returns with a config files according to the given URI
+
+## Working with HTTPS
+
+This config provider also has support to HTTPS. By default, It will rely on PKI in order to stablish the connection between the client and the server. It also provides mechanisms for you to specicy your own certificate authoritiy as well as bypass hostname validation.
```

Comment:
```suggestion
This config provider also has support to HTTPS. By default, It will rely on PKI in order to establish the connection between the client and the server. It also provides mechanisms for you to specify your own certificate authority as well as bypass hostname validation.
```